<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Geomove</class>
 <widget class="QMainWindow" name="Geomove">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>741</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geomove</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="MainWindowLayout" stretch="1,7,2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="Data" stretch="0,1,0,0,2,0">
        <item>
         <widget class="QPushButton" name="addMap">
          <property name="text">
           <string>(+)   Add Map</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listMaps">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addLocation">
          <property name="text">
           <string>(+)   Add Location</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="Filter">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="Filtervalues" stretch="1,0,1">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QDoubleSpinBox" name="agemin">
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="suffix">
               <string>Ma</string>
              </property>
              <property name="maximum">
               <double>99.989999999999995</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="Tiret">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string> - </string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="agemax">
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="suffix">
               <string>Ma</string>
              </property>
              <property name="maximum">
               <double>250.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>250.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="ConfirmationFilter">
            <property name="text">
             <string>Confirm Locations Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listLocations">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Zoom">
          <item>
           <widget class="QPushButton" name="Zoomout">
            <property name="text">
             <string>Zoom (-)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Zoomin">
            <property name="text">
             <string>Zoom (+)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="MapScreen">
        <item>
         <widget class="QGraphicsView" name="map"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="Characteristics">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QListView" name="characteristics">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionRectVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_Map"/>
    <addaction name="actionAdd_coordinates"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Edition">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="actionImport_locations"/>
    <addaction name="actionImport_GSL"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Selected_Location"/>
    <addaction name="separator"/>
    <addaction name="actionExportMapPng"/>
    <addaction name="action_ExportMaptxt"/>
    <addaction name="actionExportMapExcel"/>
   </widget>
   <widget class="QMenu" name="menu_Settings">
    <property name="title">
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionMVMScale"/>
    <addaction name="actionDisplay_scale"/>
    <addaction name="actionExport_scale"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help ?</string>
    </property>
    <addaction name="actionTuto"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edition"/>
   <addaction name="menu_Settings"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionNew_Map">
   <property name="text">
    <string>New Map</string>
   </property>
  </action>
  <action name="action_Open_Map">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Open Map</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Open recent Map</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as ...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionChange_language">
   <property name="text">
    <string>Change language</string>
   </property>
  </action>
  <action name="actionImport_locations">
   <property name="text">
    <string>Import Locations from Excel</string>
   </property>
  </action>
  <action name="actionChange_scale">
   <property name="text">
    <string>Change scale</string>
   </property>
  </action>
  <action name="actionTuto">
   <property name="text">
    <string>Watch didacticiel</string>
   </property>
  </action>
  <action name="actionImport_GSL">
   <property name="text">
    <string>Import Global Sea Level from Excel</string>
   </property>
  </action>
  <action name="actionDelete_Selected_Location">
   <property name="text">
    <string>Delete Selected Location</string>
   </property>
  </action>
  <action name="action_Export_Map_as_PNG">
   <property name="text">
    <string>&amp;Export Map as PNG</string>
   </property>
  </action>
  <action name="actionExportMapPng">
   <property name="text">
    <string>Export Map as png</string>
   </property>
  </action>
  <action name="action_ExportMaptxt">
   <property name="text">
    <string>Export Map data as txt</string>
   </property>
  </action>
  <action name="actionExportMapExcel">
   <property name="text">
    <string>Export Map data as Excel</string>
   </property>
  </action>
  <action name="actionMVMScale">
   <property name="text">
    <string>Set scale</string>
   </property>
  </action>
  <action name="actionDisplay_scale">
   <property name="text">
    <string>Display scale</string>
   </property>
  </action>
  <action name="actionExport_scale">
   <property name="text">
    <string>Export scale as png</string>
   </property>
  </action>
  <action name="actionAdd_coordinates">
   <property name="text">
    <string>Add geo. coordinates to this map</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
